recipe/meta.yaml

```diff
@@ -24,38 +28,35 @@ build:
     - MIGRATING={{ migrating }}
 
 requirements:
-  {% if not win %}
-  build:
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - ipython >=7.23.1                       # [build_platform != target_platform]
-    - jupyter_client >=6.1.12                # [build_platform != target_platform]
-    - packaging                              # [build_platform != target_platform]
-    - psutil                                 # [build_platform != target_platform]
-    - python                                 # [build_platform != target_platform]
-  {% endif %}
   host:
     - debugpy >=1.0.0
     - ipython >=7.23.1
     - jupyter_client >=6.1.12
     - packaging
     - pip
     - psutil
-    - python
+    - python 3.9
```

Comment:
Why does the Python version need to be pinned?

Reply:
Not sure why that specific version was picked... probably just needs to be `>= {{ min_python }}`?

Reply:
This is just to ensure that the host and test environments don't have the same version, which is important for validating the package.

Reply:
Would it make sense to build against the minimum Python version here (3.7) and then use a different version in test (3.8 or 3.9 for example)?